components/Map/Map.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+
+const style = {
+  width: '100%',
+  height: '600px'
+};
+
+interface MapProps {
+  saveMapState: (arg0: any, arg1: any) => void;
+}
+
+const Map = ({ saveMapState }: MapProps) => {
+  useEffect(() => {
+    const L = require('leaflet');
+
+    // create map
+    const map = L.map('map', {
+      center: [ 0.6976, 32.5825 ],
```

Comment:
The centre can't be the same for all maps ... should be a prop. Unless u mean this to only be use for one kind of map, say Uganda?

Reply:
Updated to a prop